app/controllers/pricing_controller.rb

```diff
@@ -2,14 +2,32 @@ class PricingController < ApplicationController
   layout "admin"
   before_action :authenticate_user!
 
+  def usage
+    # Same as show but renders usage view
+    @account = current_user.account
+    @current_plan_label = "Current plan"
```

Comment:
This variable @current_plan_label is used only in this controller and only a string value is assigned to it, it might be better to use a constant so as not to assign the same text to the variable